datasets/breheny.py

```diff
@@ -0,0 +1,59 @@
+from benchopt import BaseDataset, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import os
+
+    import appdirs
+    import numpy as np
+    from download import download
+    from rpy2 import robjects
+    from rpy2.robjects import numpy2ri
+    from scipy.sparse import csc_array
+
+
+def fetch_breheny(dataset: str):
+    base_dir = appdirs.user_cache_dir("benchmark_lasso_path")
```

Comment:
Could we store all those files in a common folder on the computer ? So that data does not need to be redownloaded across benchmarks. 

WDYT of adding them to the `libsvmdata` package instead?

Reply:
This should be ok now, I set the base folder to `Path.home() / 'data' / 'libsvm'`.